src/events/processCollectedSignatures/index.js

```diff
@@ -46,11 +47,12 @@ function processCollectedSignaturesBuilder(config) {
           NumberOfCollectedSignatures
         } = colSignature.returnValues
 
+        const logger = rootLogger.child({
+          eventTransactionHash: colSignature.transactionHash
+        })
+
         if (authorityResponsibleForRelay === web3Home.utils.toChecksumAddress(VALIDATOR_ADDRESS)) {
-          logger.info(
-            { eventTransactionHash: colSignature.transactionHash },
-            `Processing CollectedSignatures ${colSignature.transactionHash}`
-          )
+          logger.info(`Processing CollectedSignatures ${colSignature.transactionHash}`)
           const message = await homeBridge.methods.message(messageHash).call()
```

Comment:
consider to add logging after this to check that we don't stuck here. The same is for the method to get signatures below.